settings/src/main/scala/org/typelevel/sbt/TypelevelSettingsPlugin.scala

```diff
@@ -149,8 +153,8 @@ object TypelevelSettingsPlugin extends AutoPlugin {
         case V(V(3, _, _, _)) =>
           warningsDotty
 
-        case V(V(2, minor, _, _)) if minor >= 13 =>
-          (warnings211 ++ warnings212 ++ warnings213 ++ warningsNsc)
+        case V(V(2, minor, patch, _)) if minor >= 13 =>
+          (warnings211 ++ warnings212 ++ warnings213(patch.getOrElse(0)) ++ warningsNsc)
```

Comment:
under what circumstances is the `0` generated here?

Reply:
should be never. the issue is we are reusing the `V` parser which is also used for parsing base versions that don't have patches.

Reply:
I'd be _tempted_ to crash explicitly instead, since this should never happen

Reply:
For consistency with #643, perhaps we should pass `Option[Int]` into `warnings213` and use `patch.forall(_ >= 12)`